requirements.txt

```diff
@@ -1,3 +1,4 @@
 paramiko==2.7.2
 python-dateutil~=2.8.1
-PyYAML==5.4
\ No newline at end of file
+PyYAML==5.4
+paho-mqtt==1.5.1
```

Comment:
I think we don't want to add this as a requirement. It's a feature only 1% or less of users will use.
Maybe we can use [this method of optional imports](https://stackoverflow.com/a/563060) and specify additional requirements required for this integration in the `INTEGRATIONS.md`.